docs/commands.md

```diff
@@ -26,6 +26,7 @@ improve it.
 |heap                      | Base command to get information about the Glibc heap structure.|
 |heap-analysis-helper      | Tracks dynamic heap allocation through malloc/free to try to detect heap vulnerabilities.|
 |hexdump                   | Display arranged hexdump (according to architecture endianness) of memory range.|
+|highlight                 | Highlight text using custom matches.|
```

Comment:
Great job on documenting correctly.

You're just missing the new entry in `.mkdocs.yml` in the root, to make ReadTheDocs generate a proper ToC.

Reply:
added